src/components/NotificationCell/NotificationCell.tsx

```diff
@@ -75,7 +75,9 @@ export const NotificationCell = React.forwardRef<
 
         {renderNodeOrFallback(
           avatar,
-          actor && <Avatar name={actor.name} src={actor.avatar} />
+          actor && ("name" in actor) && !!actor.name && (
```

Comment:
I think we can do just `actor.name` here given it's a string without the `!!` as it will be truthy?

Reply:
Unfortunately we cannot. The TS compiler will raise an error if we even try to access `.name` on the union since it cannot be guaranteed to exist.

Reply:
even with the `name in actor` to do type narrowing? for clarity i'm suggesting: `actor && "name" in actor && actor.name && (`

Reply:
OH yea that will work!

Reply:
Pushed the change.